ApplicationLibCode/ProjectDataModel/StimPlanModel/RimFaciesInitialPressureConfig.cpp

```diff
@@ -38,7 +38,12 @@ RimFaciesInitialPressureConfig::RimFaciesInitialPressureConfig()
     m_faciesValue.uiCapability()->setUiHidden( true );
 
     // Use unicode for delta letter
+#ifdef Q_OS_WIN
```

Comment:
It is possible to use html tags also, these are a bit more readable.
https://github.com/OPM/ResInsight/blob/e2fc4e3de6137035bf38df8f91e8878bc80a5bba/ApplicationLibCode/ProjectDataModel/Completions/RimFractureTemplate.cpp#L193

Reply:
Tried using html, but it does not work as expected for the the table header. The raw HTML is shown.